plugins/xcmetrics/api-report.md

```diff
@@ -0,0 +1,21 @@
+## API Report File for "@backstage/plugin-xcmetrics"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { BackstagePlugin } from '@backstage/core-plugin-api';
+import { RouteRef } from '@backstage/core-plugin-api';
+
+// @public (undocumented)
+export const XcmetricsPage: () => JSX.Element;
+
+// @public (undocumented)
+export const xcmetricsPlugin: BackstagePlugin<{
```

Comment:
Can we standardise these variable names?

`xcmetrics` or `XCMetrics`  - it's a little confusing I think 😅

Reply:
This is in the API report but I mean in the plugin in general

Reply:
I can definitely agree that it's a little confusing! After some consideration I settled for the most simple way: `xcmetrics`